themes/src/main/resources-community/theme/base/login/messages/messages_pt_BR.properties

```diff
@@ -2,42 +2,51 @@ doLogIn=Entrar
 doRegister=Cadastre-se
 doCancel=Cancelar
 doSubmit=Ok
+doBack=Voltar
 doYes=Sim
 doNo=N\u00E3o
-doContinue=Continue
+doContinue=Continuar
+doIgnore=Ignorar
 doAccept=Aceitar
 doDecline=Rejeitar
-doForgotPassword=Esqueceu sua senha?
+doForgotPassword=Esqueceu a senha?
 doClickHere=Clique aqui
 doImpersonate=Personificar
+doTryAgain=Tente novamente
+doTryAnotherWay=Tente Outra Maneira
 kerberosNotConfigured=Kerberos N\u00E3o Configurado
 kerberosNotConfiguredTitle=Kerberos N\u00E3o Configurado
 bypassKerberosDetail=Ou voc\u00EA n\u00E3o est\u00E1 logado via Kerberos ou o seu navegador n\u00E3o est\u00E1 configurado para login Kerberos. Por favor, clique em continuar para fazer o login no atrav\u00E9s de outros meios
 kerberosNotSetUp=Kerberos n\u00E3o est\u00E1 configurado. Voc\u00EA n\u00E3o pode acessar.
-registerWithTitle=Registre-se com {0}
-registerWithTitleHtml=Registre-se com <strong>{0}</strong>
+registerTitle=Registre-se
 loginTitle=Entrar em {0}
-loginTitleHtml=Entrar em <strong>{0}</strong>
-impersonateTitle={0} Impersonate User
-impersonateTitleHtml=<strong>{0}</strong> Impersonate User
-realmChoice=Realm
+loginTitleHtml={0}
+impersonateTitle={0} Personificar Usu\u00E1rio
```

Comment:
`Impersonar` seems more correct to me.

Reply:
`Impersonar` is an anglicism. The correct way would be 'Personificar'.
https://ciberduvidas.iscte-iul.pt/consultorio/perguntas/o-anglicismo-impersonar-e-imitar/22977